recipes/aws-sdk-cpp/all/conanfile.py

```diff
@@ -0,0 +1,263 @@
+            "comprehend",
+            "config",
+            "cur",
+            "datapipeline",
+            "dax",
+            "devicefarm",
+            "directconnect",
+            "discovery",
+            "dlm",
+            "dms",
+            "docdb",
+            "ds",
+            "dynamodb",
+            "dynamodbstreams",
+            "ec2",
+            "ecr",
+            "ecs",
+            "eks",
+            "elasticache",
+            "elasticbeanstalk",
+            "elasticfilesystem",
+            "elasticloadbalancing",
+            "elasticloadbalancingv2",
+            "elasticmapreduce",
+            "elastictranscoder",
+            "email",
+            "es",
+            "events",
+            "firehose",
+            "fms",
+            "fsx",
+            "gamelift",
+            "glacier",
+            "globalaccelerator",
+            "glue",
+            "greengrass",
+            "guardduty",
+            "health",
+            "iam",
+            "identity-management",
+            "importexport",
+            "inspector",
+            "iot-data",
+            "iot-jobs-data",
+            "iot",
+            "kafka",
+            "kinesis",
+            "kinesisanalytics",
+            "kinesisvideo",
+            "kms",
+            "lambda",
+            "lex",
+            "lightsail",
+            "logs",
+            "machinelearnings",
+            "macie",
+            "marketplace-entitlement",
+            "marketplacecommerceanalytics",
+            "mediaconvert",
+            "medialive",
+            "mediapackage",
+            "mediastore",
+            "mediatailor",
+            "meteringmarketplace",
+            "mobileanalytics",
+            "monitoring",
+            "mq",
+            "mturk-requester",
+            "neptune",
+            "opsworks",
+            "opsworkscm",
+            "organizations",
+            "pinpoint",
+            "polly",
+            "pricing",
+            "queues",
+            "quicksight",
+            "ram",
+            "rds",
+            "redshift",
+            "recognition",
+            "resource-groups",
+            "robomaker"
+            "route53",
+            "route53domains",
+            "s3",
+            "s3-encryption",
+            "sagemaker",
+            "sdb",
+            "serverlessrepo"
+            "servicecatalog",
+            "servicediscovery",
+            "shield",
+            "signer",
+            "sms",
+            "snowball",
+            "sns",
+            "sqs",
+            "ssm",
+            "states",
+            "storagegateway",
+            "sts",
+            "support",
+            "swf",
+            "text-to-speech",
+            "texttract",
+            "transcribe",
+            "transfer",
+            "translate",
+            "waf",
+            "workdocs",
+            "worklink",
+            "workmail",
+            "workspaces",
+            "xray"
+           )
```

Comment:
```suggestion
    sdks = ("access-management",
            "accessanalyzer",
            "acm",
            "acm-pca",
            "alexaforbusiness",
            "amp",
            "amplify",
            "amplifybackend",
            "apigateway",
            "apigatewaymanagementapi",
            "apigatewayv2",
            "appconfig",
            "appflow",
            "appintegrations",
            "application-autoscaling",
            "application-insights",
            "appmesh",
            "appstream",
            "appsync",
            "athena",
            "auditmanager",
            "autoscaling",
            "autoscaling-plans",
            "awstransfer",
            "backup",
            "batch",
            "braket",
            "budgets",
            "ce",
            "chime",
            "cloud9",
            "clouddirectory",
            "cloudformation",
            "cloudfront",
            "cloudhsm",
            "cloudhsmv2",
            "cloudsearch",
            "cloudsearchdomain",
            "cloudtrail",
            "codeartifact",
            "codebuild",
            "codecommit",
            "codedeploy",
            "codeguru-reviewer",
            "codeguruprofiler",
            "codepipeline",
            "codestar",
            "codestar-connections",
            "codestar-notifications",
            "cognito-identity",
            "cognito-idp",
            "cognito-sync",
            "comprehend",
            "comprehendmedical",
            "compute-optimizer",
            "config",
            "connect",
            "connect-contact-lens",
            "connectparticipant",
            "cur",
            "customer-profiles",
            "databrew",
            "dataexchange",
            "datapipeline",
            "datasync",
            "dax",
            "detective",
            "devicefarm",
            "devops-guru",
            "directconnect",
            "discovery",
            "dlm",
            "dms",
            "docdb",
            "ds",
            "dynamodb",
            "dynamodbstreams",
            "ebs",
            "ec2",
            "ec2-instance-connect",
            "ecr",
            "ecr-public",
            "ecs",
            "eks",
            "elastic-inference",
            "elasticache",
            "elasticbeanstalk",
            "elasticfilesystem",
            "elasticloadbalancing",
            "elasticloadbalancingv2",
            "elasticmapreduce",
            "elastictranscoder",
            "email",
            "emr-containers",
            "es",
            "eventbridge",
            "events",
            "firehose",
            "fms",
            "forecast",
            "forecastquery",
            "frauddetector",
            "fsx",
            "gamelift",
            "glacier",
            "globalaccelerator",
            "glue",
            "greengrass",
            "greengrassv2",
            "groundstation",
            "guardduty",
            "health",
            "healthlake",
            "honeycode",
            "iam",
            "identity-management",
            "identitystore",
            "imagebuilder",
            "importexport",
            "inspector",
            "iot",
            "iot-data",
            "iot-jobs-data",
            "iot1click-devices",
            "iot1click-projects",
            "iotanalytics",
            "iotdeviceadvisor",
            "iotevents",
            "iotevents-data",
            "iotfleethub",
            "iotsecuretunneling",
            "iotsitewise",
            "iotthingsgraph",
            "iotwireless",
            "ivs",
            "kafka",
            "kendra",
            "kinesis",
            "kinesis-video-archived-media",
            "kinesis-video-media",
            "kinesis-video-signaling",
            "kinesisanalytics",
            "kinesisanalyticsv2",
            "kinesisvideo",
            "kms",
            "lakeformation",
            "lambda",
            "lex",
            "lex-models",
            "lexv2-models",
            "lexv2-runtime",
            "license-manager",
            "lightsail",
            "location",
            "logs",
            "lookoutvision",
            "machinelearning",
            "macie",
            "macie2",
            "managedblockchain",
            "marketplace-catalog",
            "marketplace-entitlement",
            "marketplacecommerceanalytics",
            "mediaconnect",
            "mediaconvert",
            "medialive",
            "mediapackage",
            "mediapackage-vod",
            "mediastore",
            "mediastore-data",
            "mediatailor",
            "meteringmarketplace",
            "migrationhub-config",
            "mobile",
            "mobileanalytics",
            "monitoring",
            "mq",
            "mturk-requester",
            "mwaa",
            "neptune",
            "network-firewall",
            "networkmanager",
            "opsworks",
            "opsworkscm",
            "organizations",
            "outposts",
            "personalize",
            "personalize-events",
            "personalize-runtime",
            "pi",
            "pinpoint",
            "pinpoint-email",
            "polly",
            "polly-sample",
            "pricing",
            "qldb",
            "qldb-session",
            "queues",
            "quicksight",
            "ram",
            "rds",
            "rds-data",
            "redshift",
            "redshift-data",
            "rekognition",
            "resource-groups",
            "resourcegroupstaggingapi",
            "robomaker",
            "route53",
            "route53domains",
            "route53resolver",
            "s3",
            "s3-encryption",
            "s3control",
            "s3outposts",
            "sagemaker",
            "sagemaker-a2i-runtime",
            "sagemaker-edge",
            "sagemaker-featurestore-runtime",
            "sagemaker-runtime",
            "savingsplans",
            "schemas",
            "sdb",
            "secretsmanager",
            "securityhub",
            "serverlessrepo",
            "service-quotas",
            "servicecatalog",
            "servicecatalog-appregistry",
            "servicediscovery",
            "sesv2",
            "shield",
            "signer",
            "sms",
            "sms-voice",
            "snowball",
            "sns",
            "sqs",
            "ssm",
            "sso",
            "sso-admin",
            "sso-oidc",
            "states",
            "storagegateway",
            "sts",
            "support",
            "swf",
            "synthetics",
            "text-to-speech",
            "textract",
            "timestream-query",
            "timestream-write",
            "transcribe",
            "transcribestreaming",
            "transfer",
            "translate",
            "waf",
            "waf-regional",
            "wafv2",
            "wellarchitected",
            "workdocs",
            "worklink",
            "workmail",
            "workmailmessageflow",
            "workspaces",
            "xray"
           )
```

I just took all aws-sdk-coo folders, expect those with test in their name (and expect core and AWSMigrationHub)